src/platforms/alexa/apis/index.ts

```diff
@@ -4,4 +4,30 @@ export { DeviceSettings } from "./DeviceSettings";
 export { InSkillPurchase } from "./InSkillPurchase";
 export { Lists } from "./Lists";
 export { IMessageRequest, Messaging } from "./Messaging";
+export {
+  EventsBuilder,
```

Comment:
maybe we should look into typescript namespaces?

Reply:
Same as my last comment